lua/deferred-clipboard.lua

```diff
@@ -7,17 +7,12 @@ local function is_continuous_clipboard_sync_enabled()
         and vim.o.clipboard ~= ''
 end
 
-local function schedule_disable_of_continuous_clipboard_sync_on_focus_change()
-    vim.api.nvim_create_autocmd({
-        'FocusGained',
-        'FocusLost',
-    }, {
-        once = true,
-        pattern = '*',
-        callback = function()
-            vim.o.clipboard = nil
-        end,
-    })
+local function disable_continuous_clipboard_sync()
```

Comment:
This function does more than its name suggests and should be removed and maybe replaced with something like `are_focus_change_events_supported()`.

If we want to keep function `disable_continuous_clipboard_sync()` it should probably only contain `vim.o.clipboard = nil`.